tests/article.ratings.test.js

```diff
@@ -1,4 +1,4 @@
-import chai from 'chai';
+import chai, { expect } from 'chai';
```

Comment:
@nkpremices can you add a test for the rating stats fix?

Reply:
I don't know how I can add a test for that as I'm just rounding up @MCFrank16 
Give me an idea please

Reply:
But I'm going to try